frontend/src/api/api.tsx

```diff
@@ -62,8 +31,8 @@ export const runSimulation = async (formConfig: FormConfig, selectedApi: string)
                 Authorization: `Bearer ${token}`,
             },
             body: JSON.stringify({
-                concs: absoluteConcentrations,
-                settings: settings,
+                concs: concentrations,
+                settings: parameters,
```

Comment:
could we not rename this as well? We are naming them `parameters` in the front-end. And also `parameters` in the adapters. It's technically only in the interface of the backend api that we are naming it `settings`. Let's do ourselves a favour and rename to `parameters` in the web api (so `RunRequest`)